README.md

```diff
@@ -37,92 +51,56 @@ cd app-frontend-react
+To start the app locally:
 
-```html
-<link
-  rel="stylesheet"
-  type="text/css"
-  href="https://altinncdn.no/toolkits/altinn-app-frontend/3/altinn-app-frontend.css"
-/>
-...
-<script src="https://altinncdn.no/toolkits/altinn-app-frontend/3/altinn-app-frontend.js"></script>
-```
+1. Clone the [Altinn Studio repository](https://github.com/Altinn/altinn-studio)
+2. Follow the steps in the [LOCALAPP.md documentation](https://github.com/Altinn/altinn-studio/blob/master/LOCALAPP.md)
+3. Follow our documentation on [how to use the local app-frontend](https://docs.altinn.studio/app/testing/local/debug/#using-other-frontend-versions) when running locally
 
-to
+## Automated tests
 
-```html
-<link
-  rel="stylesheet"
-  type="text/css"
-  href="http://localhost:8080/altinn-app-frontend.css"
-/>
-...
-<script src="http://localhost:8080/altinn-app-frontend.js"></script>
-```
+### End to end tests
 
-This will make the browser request the files from the local development server.
+End to end tests are using Cypress, see [test readme for how to run these](./cypress/README.md).
 
-In addition, you need to serve the app from somewhere. There are two ways of doing this, either deploy the application via Altinn Studio, or run the app locally on your machine.
+### Unit tests
 
-#### Using applications that are deployed via Altinn Studio
+Unit tests are written using Jest and React testing library
 
-After you make the changes to the `views/Home/Index.cshtml` file as mentioned above, you need to [deploy the application](https://docs.altinn.studio/app/deployment/). When accessing the application, it should now be using the app-frontend code that is served from `http://localhost:8080`.
+1. Execute `yarn --immutable`. This step is only nescessary if you have not already done it, or when package.json changes.
+2. Execute `yarn run test`.
 
-#### Using apps running locally on your machine
+## Contributing
 
-If you prefer to run the application locally (useful if you also want to make rapid changes to the application itself), you also need to clone [Altinn Studio repository](https://github.com/Altinn/altinn-studio), and follow the steps in the [LOCALAPP.md documentation](https://github.com/Altinn/altinn-studio/blob/master/LOCALAPP.md).
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
 
-## Code style
+### Code style
 
 We use [eslint](https://eslint.org/) and [prettier](https://prettier.io/), and automatically set up git hooks to enforce
-these on commits. To avoid confusion, it is recommended to set this up your IDE.
-
-### Visual Studio Code
+these on commits. To avoid confusion, it is recommended to set this up your IDE:
 
+<details>
+<summary>Visual Studio Code</summary>
 Install the [eslint extension from the marketplace](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+</details>
```

Comment:
Maybe also mention the prettier extension?
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

There is also an `extensions.json` file that suggests recommended extensions for the workspace, maybe this should be updated as well after moving since c-sharp is not really useful in this repo.

Reply:
Updated this file here: https://github.com/Altinn/app-frontend-react/pull/727
The readme could also say to simply install the recommended extensions.

Reply:
Eslint is set up to run prettier automatically, so I don't think you need the prettier extension? That will just do the job twice, and possibly cause problems for eslint (as we have set up eslint to override stuff from prettier if we want to).

Reply:
I don't really use vscode myself, but if the recommended extensions pop up when opening the project, then yes, I guess we could just write that? :shrug: Could you formulate something meaningful for vscode users (feel free to commit to this branch)?

Reply:
It is possible that the prettier extension is not needed, actually not sure how it works